flink-runtime-web/web-dashboard/src/app/pages/job/overview/chart/job-overview-drawer-chart.component.ts

```diff
@@ -64,15 +71,35 @@ export class JobOverviewDrawerChartComponent implements OnInit, OnDestroy {
   closeMetric(metric: string) {
     this.listOfSelectedMetric = this.listOfSelectedMetric.filter(item => item !== metric);
     this.jobService.metricsCacheMap.set(this.cacheMetricKey, this.listOfSelectedMetric);
+    this.showList = [metric,...this.showList];
     this.updateUnselectedMetricList();
   }
 
   updateUnselectedMetricList() {
     this.listOfUnselectedMetric = this.listOfMetricName.filter(item => this.listOfSelectedMetric.indexOf(item) === -1);
+    this.showList = this.showList.filter(item => this.listOfSelectedMetric.indexOf(item) === -1);
+    this.optionList = this.listOfUnselectedMetric;
   }
 
   constructor(private metricsService: MetricsService, private jobService: JobService, private cdr: ChangeDetectorRef) {}
 
+  nzOnSearch(val: string){
+    this.showListPageNum=0;
```

Comment:
```suggestion
    this.showListPageNum = 0;
```
Hi @faaronzheng 
thanks a lot for your PR.
it seems that the code is not formatted, there is a pre-commit hooks in the package.json it seems that it does not work in your PR, have you install the node_modules when create the pull request?

Reply:
Hi @vthinkxie 
Thanks for your review, I am not sure what kind of format should be used. Is reformate code in IDEA ok? I haven't install node_modules when create this PR, it will be nice if you can say it more clearly.

Reply:
there will be a pre commit hook after you install the node_modules, it will format the code automaticlly

Reply:
thanks, the code is already formatted by pre-commit hooks.